python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -175,6 +185,16 @@
         is_required=False,
         description="Additional sidecar containers to include in code server task definitions.",
     ),
+    "server_ecs_tags": Field(
+        Array(
+            {
+                "key": Field(StringSource, is_required=True),
+                "value": Field(StringSource, is_required=False),
+            }
+        ),
+        is_required=False,
+        description="Additional tags to apply to the launched ECS task for a code server.",
+    ),
```

Comment:
not actually used anywhere in OSS yet, but will in an exciting future where we are directly managing code servers within an agent or daemon (and needs to be included here b/c EcsContainerContext is used in both OSS and cloud)